### ISSUE TITLE
Code Generator Registration das not work when using current NuGet Packages

### ISSUE THREAD

**Issue by Jan-Ka** [madskristensen/SingleFileGeneratorSample#1]
Hi,

I found that updating the required NuGet Packages to current versions (or just following the Tutorial) does not end with a working Implementation. If I download this repository and run it as is - everything works. Downloading this repository and updating the NuGet Packages via the _Manage Packages_ Dialog also breaks the Extension.

When using the Context Action an ActivityLog is created stating that the Assembly containing the AsyncPackage cannot be found?

Not sure whats up with that and the documentation is severely lacking.

Thank you for your time.

**Comment by stephenquan** [comment]
Agreed. This project definitely could benefit with updated NuGet packages.
